src/aioamazondevices/login.py

```
@@ -0,0 +1,445 @@
+        _LOGGER.debug("Register at %s", login_url)
+        login_soup, _ = await self._http_wrapper.session_request(
+            method=login_method,
+            url=login_url,
+            input_data=login_inputs,
+        )
+
+        if not login_soup.find("input", id="auth-mfa-otpcode"):
+            _LOGGER.debug(
+                'Cannot find "auth-mfa-otpcode" in html source [%s]', login_url
+            )
+            raise CannotAuthenticate("MFA OTP code not found on login page")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)